admin/app/components/solidus_admin/sidebar/account_nav/component.html.erb

```diff
@@ -0,0 +1,38 @@
+<nav data-controller="<%= stimulus_id %>" class="mt-auto w-48">
+  <ul data-<%= stimulus_id %>-target="links"
+    class="
+      hidden
+      p-2
+      body-small text-black
+      border border-gray-100 rounded-lg shadow-base
+    ">
+    <li class="h-8 flex items-center hover:bg-gray-25">
+      <%= link_to @account_path do %>
+        <%= remixicon("user-3-line", class: "w-[0.83rem] h-[1.09rem] mx-2 fill-black") %>
+        <%= t('.account.account') %>
+      <% end %>
+    </li>
+    <li class="h-8 flex items-center hover:bg-gray-25">
+      <%= link_to @logout_path, method: :delete do %>
```

Comment:
The `method` used for logout tends to be configurable, you think we can pass it from outside?